extensions/shared/register-jetpack-block.js

```diff
@@ -57,11 +57,10 @@ function buildBlockTags( name ) {
  * @returns {string} Block title
  */
 function buildBlockTitle( blockTitle, blockTags = [] ) {
-	if ( blockTags.length ) {
-		blockTitle = `${ blockTitle } (${ blockTags.join( ', ' ) })`;
+	if ( ! blockTags?.length ) {
```

Comment:
Nitpick... `blockTags` has a default value, so there's no need for `?`.

Reply:
It doesn't. Good 👁️ !. Let me address it in a follow-up change. I don't wanna lose the current approbation, because it doesn't hurt using the `?`, right?

Reply:
solved here 96601c59ea5aa0310425ec3e450861be2aa5b944